test/transcoding/SPV_INTEL_memory_access_aliasing/intel-alias-empty-md.ll

```diff
@@ -0,0 +1,41 @@
+; Check whether the translator fails on translation of empty aliasing metadata
+; or not
```

Comment:
NIT: I think we all agree that translator shouldn't fail at all :)
```suggestion
; Check that the translator doesn't fail on translation of empty aliasing metadata
```

Reply:
Indeed :)